src/lib/mina_lib/mina_lib.ml

```diff
@@ -1410,9 +1410,11 @@ let create ?wallets (config : Config.t) =
               (fun () ->
                 O1trace.thread "manage_prover_subprocess" (fun () ->
                     Prover.create ~logger:config.logger
+                      ~enable_internal_tracing:(Internal_tracing.is_enabled ())
+                      ~internal_trace_filename:"prover-internal-trace.jsonl"
```

Comment:
Why `prover-internal-trace.json1`? Could this be `.json` instead?

Reply:
`.jsonl` was used because the file itself is a sequence of JSON lines as per https://jsonlines.org/ (which suggests `.jsonl` as a convention), but cannot in itself be parsed as a single json. But I don't mind using `.json` (or even `.log`).

Reply:
Makes sense, thanks for clarifying!